danuluma/dannstore
# #161214122 add endpoint to retrieve single sale record

#### What does this PR do?
Add endpoint to retrieve a sale record by its id

#### Description of Task to be completed?
Add the endpoint ```GET /sales/<saleId>```
Write its tests

#### How should this be manually tested?
* Set up the app locally as described in the installation documentation
* Use postman to send a POST request to http://localhost:5000/dann/api/v1/sales 
* The POST request should have just the book id in the body with the format of  ```{
        'book_id': int
    }```
* You may now send a GET request to http://localhost:5000/dann/api/v1/sales/1
* You should get a response containing the details of the sale record created previously

#### What are the relevant pivotal tracker stories?
[#161214122](https://www.pivotaltracker.com/story/show/161214122)
